tests/roots/test-warnings/includes/shared-hyperlinks.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+.. _ambiguous_shared_hyperlink:
+
+This hyperlink target is declared once and is included by multiple documents.
```

Comment:
@AA-Turner @picnixz is there a way to determine a unique declaration identifier/path -- e.g. where a hyperlink was defined, instead of where it is used?

When reporting warnings about duplicated anonymous hyperlinks, it'd be nice to be able to indicate clearly that the duplication originates from multiple `include`s of a common document, instead of weird/confusing warnings about the hyperlinks appearing in multiple documents.